common/python/database/lmdb_helper_proxy.py

```diff
@@ -168,6 +140,124 @@ def lookup(self, table):
             logger.error("Unknown response format")
         return result
 
+# ------------------------------------------------------------------------------
+    def csv_append(self, table, key, value):
+        """
+        Function to update a key-value pair in a lmdb table that holds
+        comma-separated strings as value. This function adds a string
+        to the end of the comma-separated value.
+
+        Parameters:
+           @param table - Name of the lmdb table in which key-value pair
+                          needs to be updated.
+           @param key - The primary key of the table.
+           @param value - The value that needs to be appended to the existing
+                          value(comma-separated) corresponding to the key.
+        """
+        # Csv_append, table, key, value
+        request = "CA\n" + self.__escape(table) + "\n" + self.__escape(key) + \
```

Comment:
Can you please add a code comment for the format of data. Does "CA" mean "CSV Append" ?